pkg/controller/networkpolicy/endpoint_querier_test.go

```diff
@@ -156,6 +157,9 @@ func makeControllerAndEndpointQuerier(objects ...runtime.Object) *endpointQuerie
 	querier := NewEndpointQuerier(c.NetworkPolicyController)
 	// start informers and run controller
 	c.informerFactory.Start(stopCh)
+	c.crdInformerFactory.Start(stopCh)
+	go c.groupingController.Run(stopCh)
+	go c.groupingInterface.Run(stopCh)
```

Comment:
Why we don't have these before and add them now?

Reply:
A complete workflow needs to start crdInformerFactory, groupingController, groupingInterface, otherwise workers wouldn't run.
Previously it can work because event handlers could also create internalNetworkPolicies and groups, now that event handlers only trigger workers, we need to ensure workers running.

Reply:
Got it. Thanks.